inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.hpp"
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <memory>
+#include <vector>
+
+namespace vpu {
+
+ConvertExtractImagePatchesToReorgYoloVPU::ConvertExtractImagePatchesToReorgYoloVPU() {
```

Comment:
Nitpick: I think `VPU` postfix is excessive there, `namespace vpu` is already point us that it is used for VPU.

Reply:
Good catch! Thx! Done